manifests/s/SanfordLP/DYMOConnect/1.4.3.131/SanfordLP.DYMOConnect.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SanfordLP.DYMOConnect
+PackageVersion: 1.4.3.131
+PackageLocale: en-US
+Publisher: Sanford L.P.

```

```suggestion
Publisher: DYMO
```
Looks like Publisher got updated so this needs to match Control Panel.

---

Agreed